src/lib/teuthologyAPI.ts

```diff
@@ -0,0 +1,18 @@
+
+const TEUTHOLOGY_API_SERVER = 
+    import.meta.env.VITE_TEUTHOLOGY_API || "";
+
+function login() {
+    const url =  new URL("/login/", TEUTHOLOGY_API_SERVER).href;
+    window.location.replace(url);
```

Comment:
Looks like `useNavigate` doesn't handle external links, using that would redirect to `http://localhost:8081/http://localhost:8082/login/`. Maybe we can go with one of the followings to not break the back button functionality:
1. window.location.href = url;
2. anchor tag

Reply:
You're right! Using `.href` instead of `.replace()` is likely good enough. Wrapping the `Button` in a `Link` could work as well; not sure if there is a reason to prefer one over the other in this case.

Reply:
Only reason I went with `.href` was to extract the logic to a custom hook instead of putting it in the Logic component. It also helps as we can do extra work before redirecting, like deleting `GH_USER` cookie before redirecting to /logout.